website/content/docs/secrets/kv/kv-v2.mdx

```diff
@@ -185,7 +185,7 @@ real path).
    version          1
    ```
 
-1. Read arbitrary data:
+2. Read arbitrary data:
```

Comment:
I believe we don't need to update the numbers here, as the markdown generation will take care of incrementing the numbers properly.

Reply:
^ You are correct, is updated to remove that.